spark/src/main/java/org/opensearch/sql/spark/execution/session/InteractiveSession.java

```diff
@@ -53,8 +53,12 @@ public void open(CreateSessionRequest createSessionRequest) {
     try {
       // append session id;
       createSessionRequest
-          .getSparkSubmitParametersBuilder()
-          .sessionExecution(sessionId.getSessionId(), createSessionRequest.getDatasourceName());
+          .getSparkSubmitParameters()
+          .acceptModifier(
+              (parameters) -> {
+                parameters.sessionExecution(
+                    sessionId.getSessionId(), createSessionRequest.getDatasourceName());
+              });
       createSessionRequest.getTags().put(SESSION_ID_TAG_KEY, sessionId.getSessionId());
```

Comment:
I don't fully understand this part..
Can't `.getSparkSubmitParameters().sessionExecution(...)` work?

Reply:
Hmm, you are right. I don't remember why I did this...